src/test/unit/target.h

```diff
@@ -17,6 +17,7 @@
 
 #pragma once
 
+#define SITL_BUILD
```

Comment:
This will probably simplify things in the long run. I think there are a few places where we check for SITL || UNITTEST